src/main/webapp/scripts/serverclient.js

```diff
@@ -40,23 +53,40 @@ class ServerClient {
      * @private {URLSearchParams}
      */
     this.urlParams_ = urlParams;
+
+    this.startSessionCache_();
   }
 
   /** 
    * This method starts to cache.
    * @private
    */
   startSessionCache_() {
-    throw new Error('Unimplemented');
+    this.sessionCache_.start();
   }
 
   /**
    * This method changes the controller of the current session to the 
-   * Attendee pased in.
-   * @param {string} name 
+   * Attendee passed in.
+   * @param {string} newControllerName 
    */
-  passController(name) {
-    throw new Error('Unimplemented');
+  changeController(newControllerName) {
```

Comment:
Last one lol change the name to changeControllerTo. I think thats even more readable.